liftwing/models/article_topic.py

```diff
@@ -0,0 +1,32 @@
+from pydantic import BaseModel, Field
+from .liftwing_model import LiftwingModel, ModelMetadata
+
+
+class ArticleTopicPayload(BaseModel):
```

Comment:
The article topic outlink model also supports optional parameters including `threshold`, `features_str`, `debug`:
https://api.wikimedia.org/wiki/Lift_Wing_API/Reference/Get_articletopic_outlink_prediction#POST_Parameters